### ISSUE TITLE
[DOC] `get_features_targets` should be removed from general_functions.rst

### ISSUE THREAD

**Issue by Zeroto521** [pyjanitor-devs/pyjanitor#891]
# Brief Description of Fix

When I see docs I found [`get_features_targets` page](https://pyjanitor-devs.github.io/pyjanitor/reference/janitor.functions/janitor.get_features_targets.html) was black.

After #556 merged, there also should remove `get_features_targets` from general_functions.rst.
But #556 didn't remove it. So this cause that page became blank.

# Relevant Context

- [`get_features_targets` page is blank](https://pyjanitor-devs.github.io/pyjanitor/reference/janitor.functions/janitor.get_features_targets.html)
- [general_functions.rst should update](https://github.com/pyjanitor-devs/pyjanitor/blob/dev/docs/reference/general_functions.rst)

**Comment by aditirao7** [comment]
Hi, could I work on this fix?

**Comment by Zeroto521** [comment]
The building tool of documentation now is moving from Sphinx to MkDocs. #894

So I'm not sure what effect would bring to this issue.

**Comment by aditirao7** [comment]
Oh okay, I'll see if I can take up another issue then. Thanks :)

**Comment by Zeroto521** [comment]
fixed.

![image](https://user-images.githubusercontent.com/25895405/138805329-52d085c6-1c5e-460c-8aaf-f4f476413c69.png)
